kano_feedback/DataSender.py

```diff
@@ -425,16 +425,16 @@ def try_connect():
     return is_internet()
 
 
-def send_form(title, body, question_id):
+def send_form(title, body, question_id, interactive=True):
     '''
     This function is used by the Feedback widget.
     The information (title, username, body, email and question id) is sent to a
     Google form.
     '''
-    if not try_connect() or not try_login():
-        KanoDialog('Unable to send',
-                   'Please check that you have internet and ' +
-                   'are logged into Kano World.').run()
+
+    if interactive and not try_connect() or not try_login():
```

Comment:
hey Albert, 

I don't think the user needs to be notified about this. Let's do it as if was sent, because it's going to be sent automatically anyways at some point